pkg/cli/doctor.go

```diff
@@ -325,6 +325,10 @@ func fromZipDir(
 
 	maybePrint := func(fileName string) string {
 		path := path.Join(zipDirPath, fileName)
+		_, err := os.Stat(path + ".err.txt")
+		if err == nil {
```

Comment:
I think you want `if err != oserror.ErrNotFound` and also error out if there's another error.